aedttest/aedt_test_runner.py

```diff
@@ -394,6 +394,8 @@ def prepare_project_report(self, project_name: str, project_path: str) -> Dict[s
 
         """
         report_file = Path(project_path).parent / f"{project_name}.json"
+        if not report_file.exists():
+            report_file = Path(project_path).parent / f"{project_name}_parasolid.json"
```

Comment:
why is that ?
we should not bundle with different kernels. If project gets automatically converted we should not decide for it.

there is an environment variable that forces proect override. Ask Ivonne, I gave it to her

Reply:
Let me ask her, she didn't tell me anything

Reply:
I found the environment variable, what do you suggest to add it to the code? Thanks!
@beliaev-maksim